llvm/include/llvm/DebugInfo/DWARF/DWARFVerifier.h

```diff
@@ -348,6 +363,9 @@ class DWARFVerifier {
   bool verifyDebugStrOffsets(
       StringRef SectionName, const DWARFSection &Section, StringRef StrData,
       void (DWARFObject::*)(function_ref<void(const DWARFSection &)>) const);
+
+  /// Emits any aggregate information collected, depending on the dump options
+  void summarize(bool Success);
```

Comment:
If we have this aggregator that gets called back on every error anyway - would that allow us to remove the `Success` returns and tracking, and just track success in the aggregator instead?

Reply:
Yeah. I think the Success boolean was a holdover from an earlier design. I'll expose a couple size attributes on the aggregator class.